corehq/apps/accounting/static/accounting/js/confirm_plan.js

```diff
@@ -45,6 +48,19 @@ hqDefine('accounting/js/confirm_plan', [
         self.otherSelected = ko.computed(function () {
             return self.newToolReason() === "Other";
         });
+        self.requiredQuestionsAnswered = ko.computed(function () {
+            if (self.downgradeReason() == null) {
```

Comment:
I'd lean towards making this `if (!self.downgradeReason())` to protect against code changes that make downgradeReason `undefined`.